HW_5/task_3.py

```diff
@@ -0,0 +1,23 @@
+"""
+Задача 3.
+Создайте функцию генератор чисел Фибоначчи (см. Википедию).
+"""
+from time import sleep
+
+
+def fibonacci_gen():
+    x = 0
+    y = 1
```

Comment:
Такое присваивание обычно делается в одной строке
```suggestion
    x, y= 0, 1
```